packages/node_modules/@webex/widget-space/src/container.js

```diff
@@ -143,19 +145,49 @@ export class SpaceWidget extends Component {
       const callStartTime = (call?.activeParticipantsCount > 0) ? call?.startTime : null;
 
       let menuButton;
+      const {preferredWebexSite} = sparkInstance.meetings;
+      let isMeetButtonDisabled = false;
+
+      if (sparkInstance.meetings.getMeetingByType(CONSTANTS.CONVERSATION_URL, currentConversation)) {
+        isMeetButtonDisabled = false;
+      }
+      else {
+        isMeetButtonDisabled = !preferredWebexSite;
+      }
+      let errorElementSpace = null;
+
+      if (isMeetButtonDisabled) {
```

Comment:
This `if` condition doesn't seem to be required. We are rendering `errorElementSpace` conditionally anyway

Reply:
@rarajes2, we can check and remove.

Reply:
@rarajes2, Done.